### DISCUSSION TITLE
OpenAI Endpoint Compatibility in TEI Library

### DISCUSSION THREAD

**Discussion by zekrismh** [huggingface/text-embeddings-inference#71] [Ideas]
Hi TEI community,

I am reaching out to open an issue regarding a compatibility concern with the OpenAI endpoint in the TEI library.

The problem arises when using the OpenAI client to request this endpoint, resulting in a 404 response. This discrepancy is due to the OpenAI embedding endpoint being "/openai/embeddings," whereas in the TEI case, it is simply "/openai."

I appreciate your attention to this matter.

Best regards,

**Comment by OlivierDehaene** [comment]
Should be fixed by #76
